src/main/java/de/uni_potsdam/hpi/asg/common/breeze/model/xml/Components.java

```diff
@@ -51,8 +51,8 @@ public static Components readIn(String filename) {
 			JAXBContext jaxbContext = JAXBContext.newInstance(Components.class);
 			Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
 			if(filename == null || filename.equals("")) {
-				URL url = Components.class.getResource(injarfilename);
```

Comment:
I've read somewhere that the stream version works the same way no matter if the file is packed in a jar or not. Some said the URL version does not work or at least not as stable, although i didn't verified this.